app/index.html

```diff
@@ -49,8 +50,7 @@
             </div>
         </nav>
     </div>
-    <div class="container"  id="regionPageContent"></div>
-
+    <div class="page-container"  id="regionPageContent"></div>
```

Comment:
**Page-container** class is or own, defined in main.css, so far not sure why we need it, especially *position: absolute*.
We agreed not to bother with design until Ilya prepares some mockup design.